pygmt/tests/test_sph2grd.py

```diff
@@ -0,0 +1,34 @@
+"""
+Tests for sph2grd.
+"""
+import os
+
+import numpy.testing as npt
+from pygmt import sph2grd
+from pygmt.helpers import GMTTempFile
+
+
+def test_sph2grd_outgrid():
+    """
+    Test sph2grd with a set outgrid.
+    """
+    with GMTTempFile(suffix=".nc") as tmpfile:
+        result = sph2grd(
+            table="@EGM96_to_36.txt", outgrid=tmpfile.name, increment=1, region="g"
```

Comment:
Remember to add `@EGM96_to_36.txt` to the cache at https://github.com/GenericMappingTools/pygmt/blob/8eb055d2a696c59da2cbbe57b473885562aeb3a9/pygmt/helpers/testing.py#L167
```suggestion
            data="@EGM96_to_36.txt", outgrid=tmpfile.name, spacing=1, region="g"
```